beacon/schemas/alternative.py

```diff
@@ -23,3 +24,134 @@ def ga4gh_service_info_v10(row):
         'version': conf.version,
         'url': conf.service_url,
     }
+
+
+def ga4gh_phenopackets_biosamples_v10(row):
+    schema_name = 'ga4gh-phenopacket-biosample-v1.0'
+    abnormal_sample_ontology = 'EFO:0009655'
+    return {
+        'id': None, # required
+        # 'subject': None,
+        # 'phenotypic_features': None,
+        'biosamples': [{
+            'id': row['biosample_stable_id'],
+            'individual_id': row['individual_stable_id'],
+            'description': row['description'],
+            'sampled_tissue': get_sampled_tissue(row['sample_origins_ontology'], schema_name),
+            'phenotypic_features': None,
+            'taxonomy': None,
+            'individual_age_at_collection': {
+                'age': row['individual_age_at_collection'],
+            },
+            'histological_diagnosis': None,
+            'tumor_progression': {
+                    'id': row['tumor_progression_ontology'],
+                    'label': row['tumor_progression_ontology_label'],
+                },
+            'tumor_grade': {
+                    'id': row['tumor_grade_ontology'],
+                    'label': row['tumor_grade_ontology_label'],
+                },
+            'diagnostic_markers': None,
+            'procedure': {
+                'code': {
+                    'id': row['obtention_procedure_ontology'],
+                    'label': row['obtention_procedure_ontology_label'],
+                },
+                'body_site': None,
+            },
+            'hts_files': None,
+            'variants': None,
+            'is_control_sample': True if row['biosample_status_ontology'] == abnormal_sample_ontology else False,
```

Comment:
Just `row['biosample_status_ontology'] == abnormal_sample_ontology` is enough...